Makefile

```diff
@@ -32,10 +39,20 @@ install:
 	@echo "  PREFIX = ${PREFIX}"
 	@echo "  DESTDIR = ${DESTDIR}"
 	@echo ""
+
+install-python-package: $(VIRTUALENV)
 	@echo "Installing python packages"
 	@echo "################################################################################"
-	$(PIP) install $(verbose_arg) .
+	$(PIP) install --index https://pypi.apple.com/simple $(verbose_arg) .
 	@echo ""
+
+$(VIRTUALENV):
+	@echo "Creating virtualenv at $(PREFIX) "
+	@echo "################################################################################"
+	virtualenv -p python3 $(PREFIX)
```

Comment:
I think running `python3 -m venv` might be more portable?

Reply:
Sure, that seems to work